lib/client/api.go

```diff
@@ -1314,95 +1316,116 @@ func (tc *TeleportClient) Join(ctx context.Context, namespace string, sessionID
 
 // Play replays the recorded session
 func (tc *TeleportClient) Play(ctx context.Context, namespace, sessionID string) (err error) {
```

Comment:
Instead of reusing sessionID for the filename (which is confusing), introduce a new `PlayFile` API on the `TeleportClient`. I would also suggest to use an `io.Reader` as input instead of the path - so that the caller is responsible for initial I/O setup.